reference/array/functions/array-combine.xml

```diff
@@ -53,45 +53,45 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   Zwraca wynikową tablicę lub &false; jeżeli liczba elementów z obu tablic
-   nie pasuje do siebie.
+   Zwraca połączoną <type>array</type>.
```

Comment:
Pozostawiam do przemyślenia, ale osobiście tłumaczyłem to w ten sposób

```suggestion
   Zwraca połączoną tablicę (<type>array</type>).
```

W zasadzie prawdopodobnie najlepiej byłoby użyć składni `<link linkend="language.types.array">tablicę</link>`. Jest to dużo dłuższe niż `<type>array</type>`, ale niestety w j. polskim nie mamy luksusu użycia angielskiego "array" bez zaburzania odmiany i wrzucania angielskiego wyrazu w środek zdania po polsku.

Sugeruję żebyśmy wybrali jedną opcję i ją także opisali w README. Potem mogę przeszukać istniejące tłumaczenia i spróbować je uspójnić.